libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss

```diff
@@ -1,6 +1,10 @@
 @use 'libs/components/theme/src/lib/styles/mixins' as mixins;
 @use 'libs/components/theme/src/lib/styles/variables' as *;
 
+sky-text-editor.sky-margin-stacked-lg {
```

Comment:
view encapsulation is off for text editor so had to use the element instead of the :host selector